### PR TITLE
Add content to help with markdown formats

### PR DESCRIPTION
**Pull request by SueChaplain**
- update front matter instructions to show equivalent markdown values
- add image format

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
@alohr51 - please confirm that the multipage format is only AsciiDoc and I will update my PR for that. Are there any other differences you want to call out?

**Comment by alohr51** [comment]
@SueChaplain yes, the `guide-multipane` layout is only for asciidoc. Markdown has to use the `guide` layout

File: README.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ guide-category: collections
 * CLI
 
 #### Approved Categories
-* none
```
Comment (alohr51) [inline README.md]: `none` is a valid category people can use (it will go into a catch all category called "More Guides")

Comment (SueChaplain) [reply README.md]: Added back. Thought that meant there were no valid categories :-)

Comment (SueChaplain) [reply README.md]: Also qualified when multi-pane could be used.

**Pull request merged**